ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -3597,6 +3597,33 @@ let rec cmdtable_data : (string * cmd_spec) list =
       ; flags= []
       }
     )
+  ; ( "tracing-create"
+    , {
+        reqd= ["name-label"]
+      ; optn=
+          [
+            "hosts"
```

Comment:
These should be consistent with other calls
```suggestion
            "host-uuids"
```

Reply:
Resolved by [ca7c05e](https://github.com/xapi-project/xen-api/pull/4966/commits/ca7c05e791c3a3d238ed4b11da845dffd118a126)